frontend/src/components/common/pagination/pagination.tsx

```diff
@@ -0,0 +1,47 @@
+import { getAllowedClasses } from 'helpers/helpers';
+import ReactPaginate from 'react-paginate';
+import { MARGIN_PAGES_DISPLAYED, PAGE_RANGE_DISPLAYED } from './common/constants/constants';
+import styles from './styles.module.scss';
+
+type Props = {
+  pageCount: number;
+  currentPage: number;
+  onPageChange: (selected: number) => void;
+  className?: string;
+};
+
+const Pagination: React.FC<Props> = ({
+  pageCount,
+  currentPage,
+  onPageChange,
+  className,
+}) => {
+  const handlePageChange = ({ selected }: { selected: number }): void => {
+    onPageChange(selected);
+  };
+
+  return pageCount ? (
+    <div className={getAllowedClasses(className)}>
+      <ReactPaginate
+        previousLabel={''}
+        nextLabel={''}
+        breakLabel={'...'}
+        pageCount={pageCount}
+        forcePage={currentPage}
+        marginPagesDisplayed={MARGIN_PAGES_DISPLAYED}
+        pageRangeDisplayed={PAGE_RANGE_DISPLAYED}
+        onPageChange={handlePageChange}
+        containerClassName={styles.pagination}
+        previousClassName={styles.previous}
+        nextClassName={styles.next}
+        pageClassName={styles.page}
+        pageLinkClassName={styles.link}
+        activeLinkClassName={styles.linkActive}
+        breakClassName={styles.break}
+        activeClassName={'active'}
+      />
+    </div>
+  ) : null;
```

Comment:
```suggestion
  return pageCount && (
    <div className={getAllowedClasses(className)}>
      <ReactPaginate
        previousLabel={''}
        nextLabel={''}
        breakLabel={'...'}
        pageCount={pageCount}
        forcePage={currentPage}
        marginPagesDisplayed={MARGIN_PAGES_DISPLAYED}
        pageRangeDisplayed={PAGE_RANGE_DISPLAYED}
        onPageChange={handlePageChange}
        containerClassName={styles.pagination}
        previousClassName={styles.previous}
        nextClassName={styles.next}
        pageClassName={styles.page}
        pageLinkClassName={styles.link}
        activeLinkClassName={styles.linkActive}
        breakClassName={styles.break}
        activeClassName={'active'}
      />
    </div>
  );
```

Reply:
It doesn't work becouse pageCount is type number, we have error: Type '({ pageCount, currentPage, onPageChange, className, }: PropsWithChildren<Props>) => 0 | Element' is not assignable to type 'FC<Props>'.

Reply:
Got you! agreed